### DISCUSSION TITLE
2.3.0: 2.4.0 (#613)

### DISCUSSION THREAD

**Discussion by bee-san** [bee-san/RustScan#614] [Announcements]
# Features
* Udp Scanning  (#596) @BrendanGlancy
*  feat(cli): add custom dns resolver option (#587) @PsypherPunk
* 
## What’s Changed

* 2.3.0 release (#613) @bee-san
* Removed unnecessary `.clone()`s (#610) @Arvmor
* Bump log from 0.4.21 to 0.4.22 (#605) @dependabot
* Bump clap from 4.5.7 to 4.5.8 (#607) @dependabot
* Bump text_placeholder from 0.5.0 to 0.5.1 (#609) @dependabot
* fix for the target x86_64-pc-windows-gnu (#611) @madneal
* Bump actions/upload-artifact from 4.3.3 to 4.3.4 (#612) @dependabot
* Bump alpine from 3.20.0 to 3.20.1 (#604) @dependabot
* Bump docker/build-push-action from 5 to 6 (#603) @dependabot
* Bump clap from 4.5.6 to 4.5.7 (#601) @dependabot
* Bump clap from 4.5.4 to 4.5.6 (#600) @dependabot
* Bump alpine from 3.19.1 to 3.20.0 (#597) @dependabot
* Bump toml from 0.8.13 to 0.8.14 (#599) @dependabot
* ci: refactor ci/cd and extends matrix build (#598) @LeoFVO
* Update dockere in README (#595) @bee-san
* Bump toml from 0.8.12 to 0.8.13 (#592) @dependabot
* Bump itertools from 0.12.1 to 0.13.0 (#593) @dependabot
* refactor: remove structopt in favour of clap (#591) @PsypherPunk
* chore: upgrade dependencies (#590) @PsypherPunk
* refactor: move address-related functions (#585) @PsypherPunk
* Bump env_logger from 0.8.4 to 0.11.3 (#583) @dependabot
* Bump toml from 0.5.11 to 0.8.12 (#582) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/RustScan/RustScan/releases/tag/2.3.0'>2.3.0: 2.4.0 (#613)</a>.</em>

**Comment by AD2011** [comment]
.deb for aarch64 or aarch64 binaries for linux please
